core/util/duckduckgo.py

```diff
@@ -135,12 +135,15 @@ def d_js_results(self):
 		root = xpath_results[self.d_js_xpath_name['results']]
 		for i in range(len(root[self.d_js_xpath_name['results_a']])):
 			a = root[self.d_js_xpath_name['results_a']][i]
-			results.append({
-				't': root[self.d_js_xpath_name['results_title']][i].text_content(),
-				'a': a,
-				'c': self.framework.meta_search_util().make_cite(a),
-				'd': root[self.d_js_xpath_name['results_content']][i].text_content(),
-			})
+			try :
+				results.append({
+					't': root[self.d_js_xpath_name['results_title']][i].text_content(),
+					'a': a,
+					'c': self.framework.meta_search_util().make_cite(a),
+					'd': root[self.d_js_xpath_name['results_content']][i].text_content(),
+				})
+			except Exception as e:
+				pass
```

Comment:
Wouldn't it be good to define some exception here?

Reply:
I'll fix this error completely. So, no.